id.go

```diff
@@ -85,6 +84,9 @@ var machineID = readMachineID()
 // pid stores the current process id
 var pid = os.Getpid()
 
+// Nil allows to check if the xid.ID == xid.Nil
+var Nil ID
```

Comment:
I would not expose this.

Reply:
agreed, I'll update if you agree with the IsNil implementation